### DISCUSSION TITLE
doris可以设置某些表存储在ssd磁盘吗

### DISCUSSION THREAD

**Discussion by littleforce163** [apache/doris#14975] [A - General / Q&A] [answered]
doris可以设置某些表存储在ssd磁盘吗

**Comment by littleforce163** [comment]
设置某些表存在ssd盘，某些表存在hdd盘

**Comment by zy-kkk** [comment, marked as answer]
确保你的be.conf里配置了SSD路径，然后建表时增加properties参数 properties {"storage_medium" = "ssd"}，不设置此 property默认为hdd，由FE配置default_storage_medium控制，默认HDD

**Comment by littleforce163** [comment]
收到，感谢zy-kkk大神
